eventlet/support/greendns.py

```diff
@@ -513,6 +513,8 @@ def getaddrinfo(host, port, family=0, socktype=0, proto=0, flags=0):
     """
     if isinstance(host, six.string_types):
         host = host.encode('idna').decode('ascii')
+    elif isinstance(host, bytes):
```

Comment:
Should be ` elif isinstance(host, six.binary_type):` to stay consistent with similar checks and ensure to support py2/3.